core/src/impl/Kokkos_Core.cpp

```diff
@@ -512,6 +522,7 @@ void parse_command_line_arguments(int& narg, char* arg[],
 
       --kokkos-help                  : print this message
       --kokkos-disable-warnings      : disable kokkos warning messages
+      --kokkos-tune-internals        : allow Kokkos to do autotuning
```

Comment:
I keep starring at it and I am still not sure I actually understand what the option does.